Cabal/ChangeLog.md

```diff
@@ -0,0 +1,784 @@
+  * Fix 'register --global/--user'
+
+### 1.8.0.4 Duncan Coutts <duncan@haskell.org> March 2010
+  * Set dylib-install-name for dynalic libs on OSX
+  * Stricter configure check that compiler supports a package's extensions
+  * More configure-time warnings
+  * Hugs can compile Cabal lib again
+  * Default datadir now follows prefix on Windows
+  * Support for finding installed packages for hugs
+  * Cabal version macros now have proper parenthesis
+  * Reverted change to filter out deps of non-buildable components
+  * Fix for registering implace when using a specific package db
+  * Fix mismatch between $os and $arch path template variables
+  * Fix for finding ar.exe on Windows, always pick ghc's version
+  * Fix for intra-package dependencies with ghc-6.12
+
+# 1.8.0.2 Duncan Coutts <duncan@haskell.org> December 2009
+  * Support for GHC-6.12
+  * New unique installed package IDs which use a package hash
+  * Allow executables to depend on the lib within the same package
+  * Dependencies for each component apply only to that component
+    (previously applied to all the other components too)
+  * Added new known license MIT and versioned GPL and LGPL
+  * More liberal package version range syntax
+  * Package registration files are now UTF8
+  * Support for LHC and JHC-0.7.2
+  * Deprecated RecordPuns extension in favour of NamedFieldPuns
+  * Deprecated PatternSignatures extension in favor of ScopedTypeVariables
+  * New VersionRange semantic view as a sequence of intervals
+  * Improved package quality checks
+  * Minor simplification in a couple Setup.hs hooks
+  * Beginnings of a unit level testsuite using QuickCheck
+  * Various bug fixes
+  * Various internal cleanups
+
+----
+
+### 1.6.0.2 Duncan Coutts <duncan@haskell.org> February 2009
+  * New configure-time check for C headers and libraries
+  * Added language extensions present in ghc-6.10
+  * Added support for NamedFieldPuns extension in ghc-6.8
+  * Fix in configure step for ghc-6.6 on Windows
+  * Fix warnings in Path_pkgname.hs module on Windows
+  * Fix for exotic flags in ld-options field
+  * Fix for using pkg-config in a package with a lib and an executable
+  * Fix for building haddock docs for exes that use the Paths module
+  * Fix for installing header files in subdirectories
+  * Fix for the case of building profiling libs but not ordinary libs
+  * Fix read-only attribute of installed files on Windows
+  * Ignore ghc -threaded flag when profiling in ghc-6.8 and older
+
+### 1.6.0.1 Duncan Coutts <duncan@haskell.org> October 2008
+  * Export a compat function to help alex and happy
+
+# 1.6.0.0 Duncan Coutts <duncan@haskell.org> October 2008
+  * Support for ghc-6.10
+  * Source control repositories can now be specified in .cabal files
+  * Bug report URLs can be now specified in .cabal files
+  * Wildcards now allowed in data-files and extra-source-files fields
+  * New syntactic sugar for dependencies "build-depends: foo ==1.2.*"
```

Comment:
Does the star here need to be escaped so that the rest of the file is not parsed as italics?

Reply:
Looks like it's only the highlighter that is confused by this.